test/e2e/clusterclass_changes_test.go

```diff
@@ -46,3 +46,29 @@ var _ = Describe("When testing ClusterClass changes [ClusterClass]", func() {
 		})
 	})
 })
+
+var _ = Describe("When testing ClusterClass changes [vcsim] [supervisor] [PR-Blocking] [ClusterClass]", func() {
+	const specName = "clusterclass-changes" // copied from CAPI
+	Setup(specName, func(testSpecificSettingsGetter func() testSettings) {
+		capie2e.ClusterClassChangesSpec(ctx, func() capie2e.ClusterClassChangesSpecInput {
+			return capie2e.ClusterClassChangesSpecInput{
+				E2EConfig:             e2eConfig,
+				ClusterctlConfigPath:  testSpecificSettingsGetter().ClusterctlConfigPath,
+				BootstrapClusterProxy: bootstrapClusterProxy,
+				ArtifactFolder:        artifactFolder,
+				SkipCleanup:           skipCleanup,
+				Flavor:                testSpecificSettingsGetter().FlavorForMode("topology"),
+				PostNamespaceCreated:  testSpecificSettingsGetter().PostNamespaceCreatedFunc,
+				ModifyControlPlaneFields: map[string]interface{}{
+					"spec.machineTemplate.nodeDrainTimeout": "10s",
```

Comment:
q: This field is now propagated in place, should we pick another field for the sake of this test? or should we pick two fields, one propagated in place, the other not... (cc @sbueringer @chrischdi)

Reply:
Thanks for catching this! So I think we also need to update this field for the govmomi test, how about we change it as following?
- Pick another field for govmomi test
- Pick two fields for supervisor test, add another new field which will lead to rollout

Reply:
Let's hear  @sbueringer @chrischdi feedback first, we can also eventually defer this change to a follow up PR if we want to take some time to think about it

Reply:
Does it even make sense to set this field in the test when its propagated in place? I think we should pick a.diferent one only so I'm personally 👍 for

> * Pick another field for govmomi test

Reply:
It doesn't make much difference. This test doesn't wait until these properties are propagated down.

But I would pick a different field:
* changing something that leads to a Machine rollout increases test coverage basically by providing more chances for potential failure (and thus, us finding bugs)
* this field specifically is already covered by the ClusterClass rollout test anyway

I don't know why this field was chosen for CAPV, let's use the same as in core CAPI: 
```go
			ModifyControlPlaneFields: map[string]interface{}{
				"spec.kubeadmConfigSpec.verbosity": int64(4),
			},
```

Reply:
Fixed it.